packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -189,29 +213,35 @@ class _MaterialScrollbar extends RawScrollbar {
     ScrollController? controller,
     bool? isAlwaysShown,
     this.trackVisibility,
+    bool? buttonVisibility,
+    ScrollbarButtonStyles? buttonStyles,
     this.showTrackOnHover,
     this.hoverThickness,
     double? thickness,
     Radius? radius,
     ScrollNotificationPredicate? notificationPredicate,
     bool? interactive,
     ScrollbarOrientation? scrollbarOrientation,
-  }) : super(
-         key: key,
-         child: child,
-         controller: controller,
-         isAlwaysShown: isAlwaysShown,
-         thickness: thickness,
-         radius: radius,
-         fadeDuration: _kScrollbarFadeDuration,
-         timeToFade: _kScrollbarTimeToFade,
-         pressDuration: Duration.zero,
-         notificationPredicate: notificationPredicate ?? defaultScrollNotificationPredicate,
-         interactive: interactive,
-         scrollbarOrientation: scrollbarOrientation,
-       );
+  }) : _buttonVisibility = buttonVisibility,
+    _buttonStyles = buttonStyles,
+    super(
+      key: key,
+      child: child,
+      controller: controller,
+      isAlwaysShown: isAlwaysShown,
+      thickness: thickness,
+      radius: radius,
+      fadeDuration: _kScrollbarFadeDuration,
+      timeToFade: _kScrollbarTimeToFade,
+      pressDuration: Duration.zero,
+      notificationPredicate: notificationPredicate ?? defaultScrollNotificationPredicate,
+      interactive: interactive,
+      scrollbarOrientation: scrollbarOrientation,
+    );
```

Comment:
Nit: these should all move over to align with the first one
```suggestion
  }) : _buttonVisibility = buttonVisibility,
       _buttonStyles = buttonStyles,
       // -> and so on
    super(
      key: key,
      child: child,
      controller: controller,
      isAlwaysShown: isAlwaysShown,
      thickness: thickness,
      radius: radius,
      fadeDuration: _kScrollbarFadeDuration,
      timeToFade: _kScrollbarTimeToFade,
      pressDuration: Duration.zero,
      notificationPredicate: notificationPredicate ?? defaultScrollNotificationPredicate,
      interactive: interactive,
      scrollbarOrientation: scrollbarOrientation,
    );
```